tests/Unit/IO/DataImporter/Test_Tags.cpp

```diff
@@ -7,8 +7,42 @@
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
 #include "IO/DataImporter/Tags.hpp"
+#include "Options/Options.hpp"
+#include "Options/ParseOptions.hpp"
+
+namespace {
+struct ExampleDataGroup {
+  using group = importer::OptionTags::Group;
+  static std::string name() noexcept { return "ExampleData"; }
+  static constexpr OptionString help = "halp";
```

Comment:
This help string seems useless to me...isn't "Example help string" a little better?

Reply:
This string is used all over the testing code for help string that are completely irrelevant, probably for no other reason than that someone started doing it and others copied. I have to fix a typo elsewhere so I’ll also change this.

Reply:
This started as a hat tip to a former Cornell grad student, FYI :)